### PR TITLE
Add yarn version plugin

### PR DESCRIPTION
**Pull request by yorhodes**
Published all packages at `0.2.0` on npm

```
yarn plugin import version
yarn workspaces foreach --no-private --verbose --topological version minor
yarn workspaces foreach --no-private --verbose --topological npm publish --access public
```

### CODE REVIEW

**Comment by nambrot** [comment]
Oh that's cool

File: .yarn/versions/c414a53e.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+undecided:
```
Comment (nambrot) [inline .yarn/versions/c414a53e.yml:1]: What is this?

**Review by nambrot — APPROVED** [review]

**Comment by nambrot** [comment]
We don't quite do this yet, but should we update the CHANGELOG files?

**Comment by nambrot** [comment]
Created release https://github.com/abacus-network/abacus-monorepo/releases/tag/v.0.2.0

**Review by jmrossy — COMMENTED** [review]
Looks good, just wondering about the c414a53e.yml file like Nam pointed out

**Pull request merged**